### PR TITLE
Hotfix/add migration

### PR DESCRIPTION
**Pull request by C0Rs1K**
Changed incorrect usings to correct and changed namespace in curtains context

### CODE REVIEW

File: Curtains/Curtains.Infrastructure/Migrations/CurtainsContextModelSnapshot.cs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿// <auto-generated />
 using System;
-using Curtains.Infrastructure.Models;
+using Curtains.Domain.Data;
```
Comment (zd-prog) [inline Curtains/Curtains.Infrastructure/Migrations/CurtainsContextModelSnapshot.cs:3]: The name of namespace should be Curtains.Domain.Models

File: Curtains/Curtains.Infrastructure/Migrations/20220901113709_InitCreate.Designer.cs [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿// <auto-generated />
 using System;
-using Curtains.Infrastructure.Models;
+using Curtains.Domain.Data;
```
Comment (zd-prog) [inline Curtains/Curtains.Infrastructure/Migrations/20220901113709_InitCreate.Designer.cs:3]: The name of namespace should be Curtains.Domain.Models

**Review by zd-prog — CHANGES_REQUESTED** [review]

**Review by zd-prog — APPROVED** [review]

**Review by EgorAntonovich — APPROVED** [review]
Approved

**Comment by C0Rs1K** [comment]
[[BE] E-1 S-1 T-3 DB creation](https://trello.com/c/ioMnVGc3/47-be-e-1-s-1-t-3-db-creation)

**Comment by C0Rs1K** [comment]
[[BE] E-1 S-1 T-6 Create EF layer for connection between API & DB](https://trello.com/c/F1wpmUz4/49-be-e-1-s-1-t-6-create-ef-layer-for-connection-between-api-db)

**Pull request merged**